### PR TITLE
Add request logger name parameter to json logging

### PR DESCRIPTION
**Pull request by neumayer**

### CODE REVIEW

File: jsonlogging/src/main/java/org/secondbase/logging/JsonLoggerConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package org.secondbase.logging;
+
+import org.secondbase.flags.Flag;
+
+public class JsonLoggerConfiguration {
+
+    @Flag(name = "environment", description = "The environment the service runs in (test|staging|production).", required = false)
```
Comment (acidmoose) [inline jsonlogging/src/main/java/org/secondbase/logging/JsonLoggerConfiguration.java]: Long line

File: jsonlogging/src/main/java/org/secondbase/logging/JsonLoggerConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    public static String environment;
+
+    @Flag(name = "service", description = "The name of the service (testservice).", required = false)
```
Comment (acidmoose) [inline jsonlogging/src/main/java/org/secondbase/logging/JsonLoggerConfiguration.java]: required=false is default.

File: jsonlogging/src/main/java/org/secondbase/logging/JsonLoggerConfiguration.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    public static String service;
+
+    @Flag(name = "datacenter", description = "The datacenter the service runs in (dc1|dc2).", required = false)
+    public static String datacenter;
+
+    @Flag(name = "request-logger-class-name", description = "Name of the request logger class", required = false)
+    public static String requestLoggerClassName;
+}
```
Comment (acidmoose) [inline jsonlogging/src/main/java/org/secondbase/logging/JsonLoggerConfiguration.java]: This new class needs to be loaded into Flags.

**Review by acidmoose — CHANGES_REQUESTED** [review]

File: jsonlogging/src/main/java/org/secondbase/logging/JsonLoggerConfiguration.java (hunk 1 above)
Comment (neumayer) [reply jsonlogging/src/main/java/org/secondbase/logging/JsonLoggerConfiguration.java]: Done.

File: jsonlogging/src/main/java/org/secondbase/logging/JsonLoggerConfiguration.java (hunk 2 above)
Comment (neumayer) [reply jsonlogging/src/main/java/org/secondbase/logging/JsonLoggerConfiguration.java]: Done.

File: jsonlogging/src/main/java/org/secondbase/logging/JsonLoggerConfiguration.java (hunk 3 above)
Comment (neumayer) [reply jsonlogging/src/main/java/org/secondbase/logging/JsonLoggerConfiguration.java]: Yeah, I had found that myself shortly after pushing. Done.

**Review by acidmoose — APPROVED** [review]

**Pull request merged**